card.css

```diff
@@ -0,0 +1,272 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+:root {
+    --primary-color: #006D77;
+    --white: #fff;
+    --gray: #808080
+}
+
+body {
+    background-color: #f5f5f5;
+    font-family: 'Poppins', sans-serif;
+}
+
+.card {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem;
+}
+
+.basic_card, 
+.horizontal_card,
+.basic_card_text_overlap,
+.basic_card_with_header,
+.basic_card_with_btns,
+.basic_card_with_icons
+{
+    background-color: var(--white);
+    border: 1px transparent;
+    border-radius: 7px;
+    width: 21rem;
+}
+
+.card > div { 
+    margin: 1rem;
+}
+
+.card_title { 
+    margin: 1rem;
+    font-size: 1.2em;
+    margin-left: 0;
+}
+
+.card_container_horizontal{
+    display: flex;
+    align-items: flex-start;
+}
+
+.card_image img,
+.basic_card_image img
+{
```

Comment:
Dont add properties to the html component only. It can hamper code maintainability

Reply:
Same as above